File: Level_1/Term_2/Progrmming 2/Ramadan-Questions/Day_2/The solution/Java/Q_1.java
package com.company; // If you encounter a problem running the code, delete this line

import java.util.Scanner;

public class Q_1 { // If you encounter a problem running the code, change the file name to "Q_1.java"

    public static void main(String[] args) {
        int grade;
        Scanner input = new Scanner(System.in);

        // Read grade from user
        System.out.print("Please enter your grade: ");
        grade = input.nextInt();

        // Check grade using if and else if and else only
        if(grade >= 90)
            System.out.println("Excellence n_n");
        else if(grade >= 80)
            System.out.println("Very good u_u");
        else if(grade >= 70)
            System.out.println("Good h_h");
        else if(grade >= 60)
            System.out.println("Acceptable @_@");
        else
            System.out.println("Not so lucky >_<, try again ^_^");
    }
}

File: Level_1/Term_2/Progrmming 2/Ramadan-Questions/Day_2/The solution/Python/Q_1.py
# Read grade from user
grade = int(input("Please enter your grade: "))

# Check grade using if and elif(else if) and else only
if grade >= 90:
    print("Excellence n_n")
elif grade >= 80:
    print("Very good u_u")
elif grade >= 70:
    print("Good h_h")
elif grade >= 60:
    print("Acceptable @_@")
else:
    print("Not so lucky >_<, try again ^_^")
